data/cluster-autoscaler-autodiscover.yaml

```diff
@@ -140,7 +141,7 @@ spec:
     spec:
       serviceAccountName: cluster-autoscaler
       containers:
-        - image: eu.gcr.io/k8s-artifacts-prod/autoscaling/cluster-autoscaler:v1.16.6
+        - image: eu.gcr.io/k8s-artifacts-prod/autoscaling/cluster-autoscaler:v1.20.0
```

Comment:
Is the cluster autoscaler version tied to the k8s version?

Reply:
I'm not sure it is. But according to AWS documentation (https://docs.aws.amazon.com/eks/latest/userguide/cluster-autoscaler.html), it says to look for the same version of the cluster:
> 5. Open the Cluster Autoscaler releases page from GitHub in a web browser and find the latest Cluster Autoscaler version that matches the Kubernetes major and minor version of your cluster.

I just saw that version 1.21 of kubernetes is available though. Do we want to use that instead?